packages/react-ui/src/widgets/TableWidgetUI/TableWidgetUI.js

```diff
@@ -12,28 +12,28 @@ import {
 } from '@mui/material';
 
 import makeStyles from '@mui/styles/makeStyles';
+import styled from '@emotion/styled';
 
-const useStyles = makeStyles((theme) => ({
-  tableHeadCellLabel: {
-    ...theme.typography.caption,
-    color: theme.palette.text.secondary
-  },
-  tableRow: {
-    maxHeight: theme.spacing(6.5),
-    transition: 'background-color 0.25s ease',
-    '&.MuiTableRow-hover:hover': {
-      cursor: 'pointer',
-      backgroundColor: theme.palette.background.default
-    }
-  },
-  tableCell: {
+const TableHeadCellLabel = styled(TableSortLabel)(({theme}) => ({
+  ...theme.typography.caption,
+  color: theme.palette.text.secondary
+}));
+
+const TableRowStyled = styled(TableRow)(({theme}) => ({
```

Comment:
Thougths about 'Styled' suffix here @vmilan? Should we promote like in other places 'prefix': StyledTableRow, for consistency?

Reply:
I thought so, but I didn't want to be picky :)

If it's possible, yes, let's keep consistency with `styled` prefix + name, when there is no better unique name.